### ISSUE TITLE
Add support for Ubuntu 20.04

### ISSUE THREAD

**Issue by joachimmetz** [log2timeline/l2tdevtools#803]
Ubuntu 20.04 is nearing release date

* python-dev is renamed to python2-dev
  * rewrite debian/control if needed #816
* add 20.04 packages to GIFT PPA
  * ~~testing~~
  * ~~dev~~
  * stable
* add 20.04 tests to l2tbinaries
  * ~~testing~~ https://travis-ci.com/github/log2timeline/l2tbinaries/builds/162989127
  * ~~dev~~ https://travis-ci.com/github/log2timeline/l2tbinaries/builds/163014083
  * stable
* changed CI tests to use 20.04 and Python 3.8 https://github.com/log2timeline/l2tdevtools/pull/817
* update documentation https://github.com/log2timeline/plaso/pull/2924
* libyal https://github.com/libyal/libyal/issues/85

**Comment by joachimmetz** [comment]
Work around for build.py dpkg on focal:
```
prep-dpkg.sh

sed 's/python-dev/python2-dev/g;s/python-setuptools/python2-setuptools/g;' -i debian/control
```

**Comment by joachimmetz** [comment]
https://github.com/log2timeline/l2tdevtools/pull/816

**Comment by joachimmetz** [comment]
changes made to l2tbinaries tests, and tests passing on testing

**Comment by joachimmetz** [comment]
Plaso 20200430 release available for 20.04 on GIFT PPA. Closing issue
